src/modules/git_state.rs

```diff
@@ -143,10 +143,16 @@ fn describe_rebase<'a>(root: &'a PathBuf, rebase_config: &'a str) -> StateDescri
         None
     };
 
+    let (current, total) = if let Some((c, t)) = progress {
+        (Some(format!("{}", c)), Some(format!("{}", t)))
+    } else {
+        (None, None)
+    };
+
     StateDescription {
         label: rebase_config,
-        current: Some(format!("{}", progress.unwrap().0)),
-        total: Some(format!("{}", progress.unwrap().1)),
+        current,
+        total,
```

Comment:
I think this issue has already been handled with #1761

Reply:
The other PR fixes the crash, but git_state is still getting bad data in gitlinked repositories.

Reply:
Sorry I only meant those specific lines.